src/lib/components/Nav.svelte

```diff
@@ -1,4 +1,4 @@
-<div><slot name="title" /></div>
+<div class="nav-title"><slot name="title" /></div>
```

Comment:
How does this let you control visibility from outside?
Does it require `:global` or something like that? I think we should try to avoid that.
If I'm not mistaken, the proper way is to add a style which uses a CSS variable and then you can use the variable to change the style from outside.

Reply:
You're right, it requires using `:global` https://github.com/dfinity/nns-dapp/pull/4668/files#diff-243e2f8111bf54358b075a4d7f13db58aa592be9394210efe583fa413326d4faR28. It can be also solved w/ the css-variable, but to have the possibility to rewrite the css variable, it should also be somewhere outside of the `Nav` component (for example in `variables.scss`). And having `--nav-component-title-display: block;` outside of the `Nav` component sounds a bit weird to me, but I'm happy to discuss this. Unconvincing excuse: the current solution serves as a temporary state until we have the SplitContent redesign.

Reply:
I think it's good that something needs to be exposed explicitly, rather than through `:global`.
It can be very difficult to find out why something is missing if it's not explicit.
Can we please do the CSS variable?

Reply:
Let's use css-var then. `--nav-component-title-display` was added.